### ISSUE TITLE
emailed python api code cannot be copied and pasted

### ISSUE THREAD

**Issue by ferrigno** [oda-hub/dispatcher-app#148]
I copied and pasted the code in the email with title 
[ODA][done] isgri_spectrum first requested at 2021-05-31 08:21:57 job_id: db2ef7ab

from oda_api.api import DispatcherAPI
disp=DispatcherAPI(url='www.astro.unige.ch/cdci/astrooda/dispatch-data', instrument='mock')

par_dict={
"src_name": "Swift J1749.4-2807",
"RA": "267.38308333",
"DEC": "-28.13494444",
"T1": "2021-02-20T00:00:00",
"T2": "2021-03-16T00:00:00",
"T_format": "isot",
"instrument": "isgri",
"osa_version": "OSA11.0",
"radius": "8",
"max_pointings": "20",
"integral_data_rights": "all-private",
"E1_keV": "28",
"E2_keV": "80",
"product_type": "Real",
"detection_threshold": "5",
"product": "isgri_spectrum",
"catalog_selected_objects": "1,2,3,4,5,6,7,8",
"selected_catalog": "{\"cat_column_descr\":[[\"meta_ID\",\"f4\"],[\"ra\",\">f4\"],[\"dec\",\">f4\"],[\"NEW_SOURCE\",\">i2\"],[\"ISGRI_FLAG\",\" "token": "",
"user_catalog_file": null
}

data_collection = disp.get_product(**par_dict) 

but got

  File "<ipython-input-1-2aabc40aaa86>", line 22
    "selected_catalog": "{\"cat_column_descr\":[[\"meta_ID\",\"f4\"],[\"ra\",\">f4\"],[\"dec\",\">f4\"],[\"NEW_SOURCE\",\">i2\"],[\"ISGRI_FLAG\",\" "token": "",
                                                                                                                                                     ^
SyntaxError: invalid syntax

**Comment by ferrigno** [comment]
In this case, I edited the catalog that was given by the image and submitted a spectrum. 
The catalog does not appear to be correctly copied i the API code.

**Comment by burnout87** [comment]
For the email we have #149 

I also see that `user_catalog` is `null`. That should not be there.

Am I right @volodymyrss ?

**Comment by volodymyrss** [comment]
> For the email we have #149
> 
> I also see that `user_catalog` is `null`. That should not be there.
> 
> Am I right @volodymyrss ?

Yes this is fixed and is, normally, now deployed (since about 2h ago).
I presume the email is from 3h ago?

We should report version of the platform maybe.

like this:

#150

**Comment by volodymyrss** [comment]
Alright, I confirm that this is supposed to be fixed but it is indeed not working with https://github.com/oda-hub/dispatcher-app/tree/7e241ff9b4704de998205603de14db25a6b8bf90

Is this right, @burnout87 ?

**Comment by volodymyrss** [comment]
Oh but it's oda_api version/problem, not dispatcher-app. Updating.
